package.json

```diff
@@ -11,9 +12,9 @@
     "npm": ">= 6"
   },
   "scripts": {
-    "build": "webpack --mode production",
-    "commit": "git-cz",
-    "watch": "webpack -w",
+    "build": "tsc",
+    "git": "git-cz",
+    "watch": "tsc -w",
```

Comment:
I believe we can keep using `webpack` in this repo since it will do all the build work.
Also, I don't see any advantages to change `webpack` to `tsc` because `webpack` already convert `.ts` to `.js` and do some optimizations.

About `"git": "git-cz",`
`commit` make more sense than `git` since this command creates a new `git commit`
```
yarn git
// vs
yarn commit
```

Reply:
ok, I reverted those changes.